src/SparseMatrixMsg.chpl

```diff
@@ -0,0 +1,89 @@
+module SparseMatrixMsg {
+    use ServerConfig;
+
+    use MultiTypeSymbolTable;
+    use MultiTypeSymEntry;
+    use SegmentedString;
+    use ServerErrorStrings;
+
+    use ArraySetops;
+    use Indexing;
+    use RadixSortLSD;
+    use Reflection;
+    use ServerErrors;
+    use Logging;
+    use Message;
+
+    use SparseMatrix;
+
+    private config const logLevel = ServerConfig.logLevel;
+    private config const logChannel = ServerConfig.logChannel;
+    const sparseLogger = new Logger(logLevel, logChannel);
+
+    param distributed = true;
+
+    proc randomSparseMatrixMsg(cmd: string, msgArgs: borrowed MessageArgs, st: borrowed SymTab): MsgTuple throws {
+        var repMsg: string; // response message with the details of the new arr
+
+        var vName = st.nextName(); // symbol table key for resulting sparse arr
+
+        var size: int = msgArgs.get("size").getIntValue();
+        var density: real = msgArgs.get("density").getRealValue();
+        // var shape: always 2d?
+        var l: string = msgArgs.getValueOf("layout");
+        // var distributed: bool = msgArgs.getBoolValueOf("distributed"); TODO does this need to be configurable?
```

Comment:
Let's remove these comments -- arrays are always distributed in Arkouda, it does not need to be configurable.